Voice/ToolkitAPI/SampleComponent/SampleComponentHelper.js

```diff
@@ -0,0 +1,41 @@
+({
+    subscribeToVoiceToolkit: function(cmp) {
+        cmp._telephonyEventListener = $A.getCallback(this.telephonyEventListener.bind(this, cmp));
+        cmp.find('voiceToolkitApi').addTelephonyEventListener('CALL_CONNECTED', cmp._telephonyEventListener);
```

Comment:
Just a heads up, this is a subset of all the events both for brevity and because I'm still working on potential changes to transcript-related events.

Reply:
@nielsenius Thanks! Does this comment clarify it better?

Reply:
Perfect!